src/plot_cartogram.R

```diff
@@ -282,3 +282,253 @@ rm_facet_clip <- function(svg_in, file_out, width){
   magick::image_write(img, file_out)
 
 }
+
+
+# national level flow time series  - instagram versioning (slide 1)
+#' @description Compose the final plot and annotate
+#' @param file_out Filepath to save to
+#' @param plot_left The national plot to position on the left
+#' @param date_start first day of focal month
+#' @param width Desired width of output plot
+#' @param height Desired height of output plot
+#' @param color_bknd Plot background color
+national_ig <- function(file_svg, plot_left, date_start, width, height, color_bknd){
+  plot_month <- lubridate::month(date_start, label = TRUE, abbr = FALSE)
+  plot_year <- lubridate::year(date_start)
+  
+  # import fonts
+  font_legend <- 'Noto Sans Mono'
+  font_add_google(font_legend)
+  showtext_opts(dpi = 300, regular.wt = 200, bold.wt = 700)
+  showtext_auto(enable = TRUE)
+  
+  text_color <- "#444444"
+    
+  # usgs logo
+  usgs_logo <- magick::image_read('in/usgs_logo.png') %>%
+    magick::image_colorize(100, text_color)
+  
+  # streamflow title
+  title_flow <- magick::image_read('in/streamflow.png')
+  
+  plot_margin <- 0.025
+  
+  # background
+  canvas <- grid::rectGrob(
+    x = 0, y = 0, 
+    width = 16, height = 9,
+    gp = grid::gpar(fill = color_bknd, alpha = 1, col = color_bknd)
+  )
+  
+  # Restyle legend
+  plot_left <- plot_left +
+    guides(fill = guide_colorsteps(
+      title = "",
+      nrow = 1,
+      direction = 'horizontal',
+      label.position = "bottom",
+      barwidth = 12,
+      barheight = 0.6,
+      background = element_rect(fill = NA),
+      show.limits = TRUE,
+      even.steps = FALSE
+    )) +
+    theme(legend.background = element_rect(fill = NA),
+          text = element_text(family = font_legend, color = text_color, size = 6.5))
+  
+  # Extract from plot
+  plot_legend <- get_legend(plot_left)
+  
+  # compose final plot
+  ggdraw(ylim = c(0,1), 
+         xlim = c(0,1)) +
+    # a white background
+    draw_grob(canvas,
+              x = 0, y = 1,
+              height = 0.37, width = 0.37,
+              hjust = 0, vjust = 1) +
+    # national-level plot
+    draw_plot(plot_left+theme(legend.position = 'none',
+                              axis.title.x.bottom = element_text(size = 14),
+                              axis.title.x.top = element_text(size = 14),
+                              axis.text.x.bottom = element_text(size = 6),
+                              axis.text.y = element_text(size = 8)),
```

Comment:
Be careful with using many different fonts sizes. Usually I try to stick to just 2